cli/src/spend_utils.rs

```diff
@@ -102,15 +111,42 @@ where
         )?;
         Ok((message, spend))
     } else {
-        let from_balance = rpc_client
-            .get_balance_with_commitment(from_pubkey, commitment)?
-            .value;
-        let from_rent_exempt_minimum = if amount == SpendAmount::RentExempt {
-            let data = rpc_client.get_account_data(from_pubkey)?;
-            rpc_client.get_minimum_balance_for_rent_exemption(data.len())?
-        } else {
-            0
-        };
+        let account = rpc_client
+            .get_account_with_commitment(from_pubkey, commitment)?
+            .value
+            .unwrap_or_default();
```

Comment:
Please look at this change especially. (Last commit)
`unwrap_or_default()` on `get_account_with_commitment()` for accounts that do not exist should yield a balance of 0 lamports and empty data, matching what would have been returned by `get_balance_with_commitment()` and `get_account_data()`.

I think we could probably get away with removing the `with_commitment`, since the `RpcClient` should be constructed with the configured commitment, but I ultimately decided to retain the approach already in use here. We could fix in a follow-up.

Reply:
This makes sense to me, and agreed on avoiding the commitment change in this PR